src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Android/X509Pal.cs

```diff
@@ -6,6 +6,7 @@
 using System.IO;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using Microsoft.Win32.SafeHandles;
```

Comment:
I don't think any of the new code needs this namespace, or did I miss something?

Reply:
Ah, left over from before I moved SafeEcKeyHandle.